### PR TITLE
{lib}[GCCcore/11.2.0] glib-networking v2.72.1

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/glib-networking/glib-networking-2.72.1-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+description = "Network extensions for GLib"
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = ['https://gitlab.gnome.org/GNOME/glib-networking/-/archive/%(version)s/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['675a654ae55d381490c5d76b84e088f684125415fbd2d46f74dfa5522c4a1268']
+
+builddependencies = [
+    ('CMake', '3.22.1'),
```
Comment (akesandgren) [inline easybuild/easyconfigs/g/glib-networking/glib-networking-2.72.1-GCCcore-11.2.0.eb:16]: glib-networking doesn't use CMake, please remove.

**Review by akesandgren — CHANGES_REQUESTED** [review]

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/g/glib-networking/glib-networking-2.72.1-GCCcore-11.2.0.eb (hunk 1 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/g/glib-networking/glib-networking-2.72.1-GCCcore-11.2.0.eb]: Done.

File: easybuild/easyconfigs/g/glib-networking/glib-networking-2.72.1-GCCcore-11.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    ('binutils', '2.37'),
+    ('Meson', '0.58.2'),
+    ('Ninja', '1.10.2'),
+    ('pkg-config', '0.29.2'),
```
[the remark this answers is not in the document]
Comment (sassy-crick) [reply easybuild/easyconfigs/g/glib-networking/glib-networking-2.72.1-GCCcore-11.2.0.eb]: Done. Thanks for both suggestions.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @sassy-crick!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (1 easyconfigs in total)
node3151.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/dcd3aa73b33a01d43f0017915e5d13b2 for a full test report.

**Pull request merged**